cms/djangoapps/contentstore/utils.py

```diff
@@ -382,6 +383,18 @@ def get_unit_url(course_locator) -> str:
     return unit_url
 
 
+def get_taxonomy_list_url():
+    """
+    Gets course authoring microfrontend URL for taxonomy list page view.
+    """
+    taxonomy_list_url = None
+    if use_tagging_taxonomy_list_page():
+        mfe_base_url = settings.COURSE_AUTHORING_MICROFRONTEND_URL
+        if mfe_base_url:
+            taxonomy_list_url = f'{mfe_base_url}/taxonomy-list'
```

Comment:
It seems that course-authoring in master now doesn't have that route.
![Peek 2024-05-08 17-47](https://github.com/nelc/edx-platform/assets/51926076/47c6cf6c-b04a-43a1-9151-68387d576495)